{"timestamp":"2025-07-13T00:02:33.358356","level":"info","event":"DAG bundles loaded: dags-folder, example_dags","logger":"airflow.dag_processing.bundles.manager.DagBundlesManager"}
{"timestamp":"2025-07-13T00:02:33.359055","level":"info","event":"Filling up the DagBag from /opt/airflow/dags/dag_analisis.py","logger":"airflow.models.dagbag.DagBag"}
{"timestamp":"2025-07-13T00:02:33.407934Z","level":"info","event":"Task instance is in running state","chan":"stdout","logger":"task"}
{"timestamp":"2025-07-13T00:02:33.408148Z","level":"info","event":" Previous state of the Task instance: TaskInstanceState.QUEUED","chan":"stdout","logger":"task"}
{"timestamp":"2025-07-13T00:02:33.408269Z","level":"info","event":"Current task name:detectar_pico","chan":"stdout","logger":"task"}
{"timestamp":"2025-07-13T00:02:33.408379Z","level":"info","event":"Dag name:a2","chan":"stdout","logger":"task"}
{"timestamp":"2025-07-13T00:02:33.408490Z","level":"info","event":"Analizando dispositivo 1, hora: 7, vista: franja_6_8","chan":"stdout","logger":"task"}
{"timestamp":"2025-07-13T00:02:33.408604Z","level":"info","event":"No se pudo acceder a la vista franja_6_8: Execution failed on sql '","chan":"stdout","logger":"task"}
{"timestamp":"2025-07-13T00:02:33.408718Z","level":"info","event":"                SELECT value","chan":"stdout","logger":"task"}
{"timestamp":"2025-07-13T00:02:33.408827Z","level":"info","event":"                FROM franja_6_8","chan":"stdout","logger":"task"}
{"timestamp":"2025-07-13T00:02:33.408936Z","level":"info","event":"                WHERE device_id = %s","chan":"stdout","logger":"task"}
{"timestamp":"2025-07-13T00:02:33.409048Z","level":"info","event":"            ': relation \"franja_6_8\" does not exist","chan":"stdout","logger":"task"}
{"timestamp":"2025-07-13T00:02:33.409185Z","level":"info","event":"LINE 3:                 FROM franja_6_8","chan":"stdout","logger":"task"}
{"timestamp":"2025-07-13T00:02:33.409299Z","level":"info","event":"                             ^","chan":"stdout","logger":"task"}
{"timestamp":"2025-07-13T00:02:33.409408Z","level":"info","event":"","chan":"stdout","logger":"task"}
{"timestamp":"2025-07-13T00:02:33.409520Z","level":"info","event":"Task instance in success state","chan":"stdout","logger":"task"}
{"timestamp":"2025-07-13T00:02:33.409631Z","level":"info","event":" Previous state of the Task instance: TaskInstanceState.RUNNING","chan":"stdout","logger":"task"}
{"timestamp":"2025-07-13T00:02:33.409735Z","level":"info","event":"Task operator:<Task(PythonOperator): detectar_pico>","chan":"stdout","logger":"task"}
{"timestamp":"2025-07-13T00:02:33.404609","level":"warning","event":"/opt/airflow/dags/dag_analisis.py:123: UserWarning: pandas only supports SQLAlchemy connectable (engine/connection) or database string URI or sqlite3 DBAPI2 connection. Other DBAPI2 objects are not tested. Please consider using SQLAlchemy.\n  df = pd.read_sql(f\"\"\"\n","logger":"py.warnings"}
{"timestamp":"2025-07-13T00:02:33.406259","level":"info","event":"Done. Returned value was: None","logger":"airflow.task.operators.airflow.providers.standard.operators.python.PythonOperator"}
